File: customView/src/main/java/com/tangbing/admin/myfirsttestproject/MainActivity.java
package com.tangbing.admin.myfirsttestproject;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

import com.tangbing.admin.myfirsttestproject.util.ActivityUtils;

public class MainActivity extends AppCompatActivity implements View.OnClickListener{
    private Button sideslipButton;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        initView();
    }
    @Override
    public void onClick(View v) {
        switch (v.getId()){
            case R.id.sideslip:
                goSideslipList();
                break;
        }
    }
    private void goSideslipList(){
        ActivityUtils.startActivity(this,CustomListViwActivity.class);
    }
    private void initView(){
        sideslipButton=findViewById(R.id.sideslip);
        sideslipButton.setOnClickListener(this);
    }
}

File: kotlindebug/src/main/java/com/tangbing/admin/kotlindebug/MainActivity.kt
package com.tangbing.admin.kotlindebug

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.util.Log;

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        d()
    }
    /**
     *包装log.d日志
     */
    fun d() {
        Log.e("MainActivity", ("1".toInt()==1).toString());
    }
}
